site/content/docs/heartbeat-checks/_index.md

```diff
@@ -0,0 +1,91 @@
+Tags are used to create meaningful distinctions between check groups, making it easy to filter out selections of checks. 
+
+### Ping URL
+The URL on which the check is listening for pings. The job or task monitored should make an HTTP request to the ping URL once per the period configured.
+
+### Period and Grace
+**Period** defines how often you expect a ping to the ping URL. 
+
+**Grace** is the time Checkly will wait before triggering any alerts when a ping does not arrive within the set period. E.g., if you have a check that expects a ping every 60 minutes, with a grace of 10 minutes, no alarms would trigger until 70 minutes after the latest ping.
+
+Use grace to compensate for variance in your jobs.
+
+### Timer
+The check timer starts when it receives its first ping and will reset after each ping. 
+If you have a check that expects a ping every 60 minutes starting at 09:30, if it receives a ping at 10:00, it will reset the timer to expect a ping before 11:00. If the check does not receive a ping before 11:00 plus any configured grace period it will trigger any configured alerts.
+> When a check is deactivated, and activated again the timer needs to be pinged to start it, just as when first creating a check.
+
+### Ping now
+Sends a ping to the ping URL. Use this to start the check timer when a check is first created or to silence alarms.
+
+### Alerting
+By default Checkly uses your account default alert settings and channels. You can configure any of the provided [alert channels](/docs/alerting/alert-channels/#managing-alert-channels) for a heartbeat check. If we don’t provide your preferred alert method, use [webhooks](/docs/alerting/webhooks) to configure your alert flow. When configuring a check, you can choose if it should use the account default channels or a selection specific to the check. 
+
+> Note that some alerting channels, like [SMS](/docs/alerting/sms-delivery/) and [Phone call](/docs/alerting/phone-calls) are only available on our [Team and Enterprise plans](https://www.checklyhq.com/pricing#features) 
+
+
+## Reporting
+The heartbeat overview page displays a table of all your heartbeat checks. Here you can see the results of recent pings, the configured period and grace, and the availability over the last 7 days.
+
+Clicking any individual check will open the check overview page.
+
+### Check overview
+The check overview page shows the current status of the check, as well as a breakdown of recent runs and availability metrics.
+
+The summary section at the top of the page allows for time-based filtering, and displays the availability and number of alerts triggered for the given time period.
+
+Single check runs can be accessed by selecting them in the timeline, or by clicking an individual result in the list below the timeline.
```

Comment:
There is no timeline for heartbeats.

Reply:
@miliberlin : There is the graph that shows pings per the selected time period 🤔 Not sure what we should call it instead of timeline. Any ideas?

Reply:
Ah right, I guess we can call that timeline.